README.md

```diff
@@ -177,6 +177,9 @@ For IdP-initiated SSO, the user will be created if it doesn't exist, but for SP-
+| **TRIGGER.CUSTOM\_DECODE\_JWT** | A hook function to decode the user JWT. This method will be called instead of the `decode_jwt_token` default function and should return the user_model.USERNAME_FIELD. This method accepts one parameter: `token`.                                                                                                                                                                                                                                                                                             | `str`            | `None`                                                                                                                                   | `my_app.models.users.decode_custom_token`         |
+| **TRIGGER.CUSTOM\_CREATE\_JWT** | A hook function to create a custom JWT for the user. This method will be called instead of the `create_jwt_token` default function and should return the token. This method accepts one parameter: `user`.                                                                                                                                                                                                                                                                                             | `str`            | `None`                                                                                                                                   | `my_app.models.users.create_custom_token`         |
+| **TRIGGER.CUSTOM\_TOKEN\_QUERY** | A hook function to create a custom query params with the jwt for the user. This method will be called after `CUSTOM_CREATE_JWT` to populate a query and attach it to a URL; should return the query params containing the token (e.g., `?token=encoded.jwt.token`). This method accepts one parameter: `token`.                                                                                                                                                                                                                                                                                             | `str`            | `None`                                                                                                                                   | `my_app.models.users.get_custom_token_query`         |
```

Comment:
Two things:
1. After adding the other two custom hooks, the `TRIGGER.CUSTOM_TOKEN_QUERY` hook seems like it can be narrowed down to just the token parameter as a constant in settings, rather than a custom hook. Or if you have something more complex on your mind, please update the example to include that.

2. Nitpick:
```suggestion
| **TRIGGER.CUSTOM\_TOKEN\_QUERY** | A hook function to create a custom query params with the JWT for the user. This method will be called after `CUSTOM_CREATE_JWT` to populate a query and attach it to a URL; should return the query params containing the token (e.g., `?token=encoded.jwt.token`). This method accepts one parameter: `token`.                                                                                                                                                                                                                                                                                             | `str`            | `None`                                                                                                                                   | `my_app.models.users.get_custom_token_query`         |
```

Reply:
1. The custom token query could be useful if the query is different than just `?token_name=token`, for example with access and refresh tokens. I have written an example in the [readme](https://github.com/grafana/django-saml2-auth/pull/35/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R246-R264) with a query function that get the token and create a query by using other library features and create an `?access=access.token&refresh=refresh.token` query.
2. Accepted!